src/network/networkctl.c

```diff
@@ -70,6 +70,7 @@
 #include "terminal-util.h"
```

Comment:
commit title: suffix `()` for function name: `check_netns_match()`
commit message: please include the cover letter of the PR.
```
networkctl: use varlink in check_netns_match()

Use varlink to detect networkd's network namespace when executing
networkctl rather than the D-Bus interface.
```